ci-operator/step-registry/baremetal/lab/ipi/conf/pxe/baremetal-lab-ipi-conf-pxe-commands.sh

```
@@ -14,6 +14,10 @@ SSHOPTS=(-o 'ConnectTimeout=5'
   -i "${CLUSTER_PROFILE_DIR}/ssh-key")
 
 CLUSTER_NAME="$(<"${SHARED_DIR}/cluster_name")"
+
+ssh "${SSHOPTS[@]}" "root@${AUX_HOST}" \
+  "systemd-cat -t '${CLUSTER_NAME}' -p5 echo 'baremetal-lab-ipi-conf-pxe: Disabling PXE boot'"
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)